src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,25 @@
+package duke.exception;
```

Comment:
Nice to have not just DukeException but the specific exceptions!

Reply:
Thank you, I think it makes the exceptions clearer to readers too!